kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -1432,6 +1417,44 @@ impl<'tcx> GotocCtx<'tcx> {
         self.codegen_expr_to_place(p, e)
     }
 
+    // Intrinsics which encode a SIMD arithmetic operation with overflow check.
+    // We expand the overflow check because CBMC overflow operations don't accept array as
+    // argument.
+    fn codegen_simd_op_with_overflow<F: FnOnce(Expr, Expr) -> Expr, G: Fn(Expr, Expr) -> Expr>(
```

Comment:
Doc comments have 3 ///

But also: love love love seeing macros get converted to functions here! 😄